src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import okapiCurrentUser from 'fixtures/okapiCurrentUser';
+import { Form } from 'react-final-form';
+import {
+  screen,
+  cleanup,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import renderWithRouter from 'helpers/renderWithRouter';
+import DepartmentsNameEdit from './DepartmentsNameEdit';
+import '__mock__/stripesSmartComponent.mock';
+
+jest.unmock('@folio/stripes/components');
+
+const onSubmit = jest.fn();
+
+const arrayMutators = {
+  pop: jest.fn(),
+  push: jest.fn(),
+};
+
+const renderDepartmentsNameEdit = (props) => {
+  const component = () => (
+    <>
```

Comment:
No need for fragments here, we can just do `component = () => <DepartmentsNameEdit {...props} />`

Reply:
Thank you, I will remove it.